Bottom: 0cc50d4fd1a993eac5c5909ae4cb8b39a54a91e7
Top:    da29e96b17891e3beef0e58875c836481c80874a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-02 21:40:16 +0100

Add source files with localized strings to .po generation


---

diff --git a/po/POTFILES.in b/po/POTFILES.in
index 01627a2..9ef4db5 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -14,6 +14,7 @@ src/edif.c
 src/error.c
 src/file.c
 src/find.c
+src/gpcb-menu.res.h
 src/hid/batch/batch.c
 src/hid/common/hidnogui.c
 src/hid/gerber/gerber.c
@@ -30,10 +31,13 @@ src/hid/gtk/gui-log-window.c
 src/hid/gtk/gui-misc.c
 src/hid/gtk/gui-netlist-window.c
 src/hid/gtk/gui-top-window.c
+src/hid/ps/ps.c
 src/main.c
+src/misc.c
 src/move.c
 src/netlist.c
 src/parse_y.c
+src/pcb-menu.res.h
 src/polygon.c
 src/rats.c
 src/report.c
